core/ledger/kvledger/hashcheck_pvtdata.go

```diff
@@ -25,7 +25,9 @@ import (
 // to the lastBlockInBootSnapshot), the boot KV hashes in the private data store are used for
 // verifying the hashes -- in this case, the write-set of a collection is trimmed in order
 // to remove the key-values that were not present in the snapshot (most likely because, they
+//
 //	were over-written in a later version). Also, the keys that have been purged explicitly by a
+//
```

Comment:
I think you just need to remove the indent here, rather than add new lines.

Reply:
it's been added with goimports, none of these changes was done manually.

Reply:
I understand, I'm suggesting goimports was confused due to the indent, it would be better to hand edit this one in a more expected way.